libdnf/conf/config.h

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
This file should be dropped as it is automatically generated.

Reply:
Hmm, I guess so. I didn't notice that before as I have this file ignored in my personal `.gitignore_global`, probably already since started to work with the `libdnf` repo where it's also not in the ignore file. But it should be, I don't see a reason why not...